src/charm.py

```diff
@@ -103,7 +104,23 @@ def _on_config_changed(self, _event: ops.EventBase) -> None:
 
     def _on_statsd_prometheus_exporter_pebble_ready(self, _event: ops.PebbleReadyEvent) -> None:
         """Handle the statsd-prometheus-exporter-pebble-ready event."""
-        statsd_container = self.unit.get_container("statsd-prometheus-exporter")
+        container = self.unit.get_container("statsd-prometheus-exporter")
```

Comment:
Do yopu think that this should go into the observability observer? Reasoning: this seems to be related to observability and there is an observability observer already that could be a good place for this

Reply:
Gotcha, updated, thanks!